apps/dashboard/src/components/welcome/inbox-connected-guide.tsx

```diff
@@ -37,140 +70,165 @@ function showStatusToast(variant: 'success' | 'error', message: string) {
-          <div className="relative mt-8 flex gap-8 first:mt-0">
-            <div className="absolute -left-[43px] flex h-5 w-5 items-center justify-center rounded-full bg-white">
-              <RiLoader3Line className="h-4 w-4 text-neutral-300" />
+    <div className="flex flex-col pl-[72px]">
+      {/* Combined section with left content and right code snippets */}
+      <div className="relative p-8 pb-12 pt-16">
+        <div className="absolute left-1 top-0 bottom-0 w-px bg-[#eeeef0]"></div>
+        <div className="relative mt-8 flex gap-8 first:mt-0 min-w-0">
+          {/* Left side - both status sections stacked */}
+          <div className="flex w-[350px] flex-col gap-8">
+            {/* First section - Inbox connected */}
+            <div className="relative flex gap-8">
+              <div className="absolute -left-[38px] flex h-5 w-5 items-center justify-center rounded-full bg-white">
+                <RiCheckboxCircleFill className="text-success h-4 w-4" />
+              </div>
+              <div className="flex flex-col gap-3">
+                <div className="flex items-center gap-2">
+                  <span className="text-success text-sm font-medium">In-App Channel Integration Activated</span>
+                </div>
+                <p className="text-foreground-400 text-xs">
+                  You've initialized your Inbox. The last step is to make an API call to confirm everything is working.
+                </p>
+              </div>
             </div>
 
-            <div className="flex gap-2">
-              {/* Left section - existing content */}
-              <div className="flex w-[350px] flex-col gap-3">
+            {/* Second section - Waiting for trigger */}
+            <div className="relative flex gap-8">
+              <div className="absolute -left-[38px] flex h-5 w-5 items-center justify-center rounded-full bg-white">
+                {hasDetectedFirstTrigger ? (
+                  <RiCheckboxCircleFill className="text-success h-4 w-4" />
+                ) : (
+                  <RiLoader3Line className="h-4 w-4 text-primary animate-spin" />
+                )}
+              </div>
+              <div className="flex flex-col gap-3">
                 <div className="flex items-center gap-2">
-                  <span className="text-sm font-medium">Let the magic happen 🪄</span>
+                  <span className="text-sm font-medium">
+                    {hasDetectedFirstTrigger ? 'Ready to complete onboarding' : 'Waiting for your first API trigger...'}
+                  </span>
                 </div>
                 <p className="text-foreground-400 text-xs">
-                  Now, trigger a notification to see it pop up in your app! If it doesn't appear, double-check that the
-                  subscriberId matches.
+                  {hasDetectedFirstTrigger
+                    ? 'Great! We detected your API trigger. Click the button to complete your onboarding.'
```

Comment:
@20lumps I think we should prompt the user to see his triggered notification on his application bell inbox here